flowmachine/flowmachine/core/model.py

```diff
@@ -72,7 +73,9 @@ def new_f(self, *args, **kwargs):
         mr = ModelResult(self, run_args=args, run_kwargs=kwargs)
         if mr.is_stored:
             return mr
+        start_time = time.perf_counter()  # TODO: Should use process time here really
```

Comment:
If you think `time.process_time()` is more appropriate, why not switch it straight away?

In general, is there any use in us recording _both_ wall time and process time? Or is this not possible with timings that are recorded by Postgres?